src/sink/firehose.rs

```diff
@@ -55,6 +55,7 @@ impl Sink for Firehose {
             let prbi = PutRecordBatchInput {
                 delivery_stream_name: self.delivery_stream_name.clone(),
                 records: chunk.iter()
+                    .filter(|m| m.value.len() < 1_024_000)
```

Comment:
It's not documented that I could find! I put an exception from the report into the commit message. What's particularly fun is that it leaks implementation details out to the user.